src/embeddings.rs

```diff
@@ -298,3 +335,47 @@ where
 
     Ok(embeddings)
 }
+
+struct Skips<'a>(HashSet<&'a str>);
+
+impl<'a> FromPyObject<'a> for Skips<'a> {
+    fn extract(ob: &'a PyAny) -> Result<Self, PyErr> {
+        let mut set = ob
+            .len()
+            .map(|len| HashSet::with_capacity(len))
+            .unwrap_or_default();
+
+        let iter = if <PySet as PyTypeInfo>::is_instance(ob) {
+            ob.iter().unwrap()
+        } else if <PyList as PyTypeInfo>::is_instance(ob) {
+            ob.iter().unwrap()
+        } else {
+            return Err(exceptions::TypeError::py_err("Iterable expected"));
+        };
+
+        for el in iter {
+            set.insert(
+                el?.extract()
+                    .map_err(|_| exceptions::TypeError::py_err("Expected String"))?,
+            );
+        }
+        Ok(Skips(set))
+    }
+}
+
+struct Embedding<'a>(&'a PyArray1<f32>);
```

Comment:
Maybe we should call this `PyEmbedding`?

Reply:
Sure, will do that later